host_vars/dev.usegalaxy.org.au.yml

```diff
@@ -199,6 +210,17 @@ galaxy_jobconf:
               rewrite_parameters: "true"
               persistence_directory: /mnt/pulsar/files/persisted_data
               submit_native_specification: "--nodes=1 --ntasks=16 --ntasks-per-node=16 --mem=49350"
+      - id: pulsar-eu-gpu-test
```

Comment:
The yml is flattened in dev_job_conf.yml.  This block would look like
```
  pulsar-eu-gpu-test:
        runner: pulsar_eu_gpu
        jobs_directory: /mnt/shared/files/staging
        transport: curl
        remote_metadata: "false"
        default_file_action: remote_transfer
        dependency_resolution: remote
        rewrite_parameters: "true"
        persistence_directory: /mnt/shared/files/persisted_data
        submit_native_specification: "--nodes=1 --
```

Reply:
no worries. Thanks

Reply:
I made a paste error above
```
submit_native_specification: "--nodes=1 --ntasks=16 --ntasks-per-node=16 --mem=49350"
```